source/standalone/workflows/ray/isaac_ray_tune.py

```diff
@@ -0,0 +1,335 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+import argparse
+import importlib.util
+import os
+import subprocess
+import sys
+
+import isaac_ray_util
+import ray
+from ray import air, tune
+from ray.tune.search.optuna import OptunaSearch
+from ray.tune.search.repeater import Repeater
+
+"""
+This script breaks down an aggregate tuning job, as defined by a hyperparameter sweep configuration,
+into individual jobs (shell commands) to run on the GPU-enabled nodes of the cluster.
+By default, (unless combined as a sub-job in a resource-wrapped aggregate job), one worker is created
+for each GPU-enabled node in the cluster for each individual job.
+
+Each hyperparameter sweep configuration should include the workflow,
+runner arguments, and hydra arguments to vary.
+
+This assumes that all workers in a cluster are homogeneous. For heterogeneous workloads,
+create several heterogeneous clusters (with homogeneous nodes in each cluster),
+then submit several overall-cluster jobs with :file:`../submit_isaac_ray_job.py`.
+KubeRay clusters on Google GKE can be created with :file:`../launch.py`
+
+Usage:
+
+.. code-block:: bash
+
+    ./isaaclab.sh -p source/standalone/workflows/ray/isaac_ray_tune.py -h
+"""
+
+DOCKER_PREFIX = "/workspace/isaaclab/"
+BASE_DIR = os.path.expanduser("~")
+PYTHON_EXEC = "./isaaclab.sh -p"
+RL_GAMES_WORKFLOW = "source/standalone/workflows/rl_games/train.py"
+
+
+class IsaacLabTuneTrainable(tune.Trainable):
+    """The Isaac Ray Tune Trainable.
```

Comment:
NIT: Could use a better docstring, but at the minimum add "Lab" :smile: 
```suggestion
    """The Isaac Lab Ray Tune Trainable.
```